Loan_Default_Classification_Code_Review_V1.py

```diff
@@ -550,7 +551,20 @@ def histogram_alt(feature):
 y_pred = clf.predict(X_test)
 y_pred
 
-
+"""
+This _looks_ like it'll be your Evaluation section. If so, call it out
+as such. Here are a few recommendations on what to evaluate:
+- Show the most important features and discuss them
```

Comment:
Can you help me take a look at my code - I was trying to get all feature's (dummy variables) coefficient matching with the name, but the code ran into an error.

Besides coefficient, there are other method I should use to determine what are the important features?